### PR TITLE
Fix bug in sequences.data.retrieve_dataframe

### PR DESCRIPTION
**Pull request by KristianVangsnes**
## Description
Running sequences.data.retrieve_dataframe with the parameter column_external_ids returned an error.

## Checklist:
- [x] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -100,8 +103,8 @@ Changes are grouped as follows
   through Data Modelling).
 
 ## [7.54.17] - 2024-08-22
-### Added
-- [Feature Preview]  Added `ExtractorExtension` model of the Core Model.
+### Fixed
+- A bug in
```
Comment (doctrino) [inline CHANGELOG.md:107]: Why did you change this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ Changes are grouped as follows
 - `Removed` for now removed features.
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
+## [7.59.1] - 2024-09-12
+### Added
+- A bug in `client.sequences.data.retrieve_dataframe(...)`.
```
Comment (doctrino) [inline CHANGELOG.md:21]: Can you be more specific? What exception was raised in what condidion?

File: CHANGELOG.md (hunk 1 above)
Comment (haakonvt) [inline CHANGELOG.md:107]: 👀

Comment (KristianVangsnes) [reply CHANGELOG.md:107]: That was not supposed to happen🤦 fixing

File: CHANGELOG.md (hunk 2 above)
Comment (KristianVangsnes) [reply CHANGELOG.md]: What do you think about the latest change?

File: tests/tests_unit/test_api/test_sequences.py [hunk 3]
```diff
@@ -565,6 +565,14 @@ def test_retrieve_dataframe_columns_mixed_with_zero(
         )
         pd.testing.assert_frame_equal(expected_df, data)
 
+    def test_retrieve_dataframe_columns_filter(
+        self, cognite_client, mock_seq_response, mock_get_sequence_data_two_col_with_zero
+    ):
+        import pandas as pd
```
Comment (doctrino) [inline tests/tests_unit/test_api/test_sequences.py:571]: The test is marked with `dsl` so you don't need this
```suggestion
```

File: CHANGELOG.md (hunk 2 above)
Comment (doctrino) [reply CHANGELOG.md]: Looks good.

**Review by doctrino — APPROVED** [review]
Good Job!!
